test/utils/navigation.spec.ts

```diff
@@ -37,26 +37,23 @@ const mockRequestWithParams = {
 describe("NAVIGATION utils", () => {
 
   test(`getEntityBackLink returns ${config.DUE_DILIGENCE_URL} when ${WhoIsRegisteringType.AGENT} selected`, () => {
-    const entityBackLink = getEntityBackLink({ [WhoIsRegisteringKey]: WhoIsRegisteringType.AGENT });
+    const mockRequest = { query: {} } as Request;
```

Comment:
there were warnings on these tests about missing request param so added them in.